### DISCUSSION TITLE
Do this project has a LICENSE I need to follow?

### DISCUSSION THREAD

**Discussion by javierojeda94** [ExperienceLovelace/ha-floorplan#160] [Q&A] [answered]
I'd like to use this project for personal usage, however I'm also planning on using it for a smart home business I'm starting... Is it allowed? Given that this is a opensource I'm ok to follow any LICENSE that might exists but I couldn't find one.

Also, I'm not planning to have any kind of property on the project. I would mention that this is community maintained and I'd give credits to you.

Any thoughts?

**Comment by exetico** [comment, marked as answer]
Hi

Thank you for asking. I've added it to the project.

Fell free to use it for both things, but please keep credits to the `ha-floorplan` repo, and if possible, mention both Petar and I by name.

https://github.com/ExperienceLovelace/ha-floorplan/blob/master/LICENSE

Have a great evening!

**Reply by javierojeda94** [reply]
Awesome, thank you so much!
